readme.md

```diff
@@ -26,3 +26,22 @@ module.exports = {
 	]
 }
 ```
+
+## Options
+
+### exclude
+
+Specify aliases to skip adding.
```

Comment:
This should also be included in the typings file

Reply:
Added already!

Reply:
I meant the docstring.

Reply:
Sure. got it.

Reply:
```ts
import { Compiler } from "webpack"

declare interface NodePolyfillPluginOptions {
	/**
	 * Specify aliases to skip adding.
	 */
	exclude?: string[]
}

declare class NodePolyfillPlugin {
	constructor(options: NodePolyfillPluginOptions)

	apply(compiler: InstanceType<typeof Compiler>): void
}
```